dragonfly-dfclient/templates/daemonset.yaml

```diff
@@ -23,6 +23,10 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+          {{- range $key, $value := .Values.dfdaemonArgs }}
+            - --{{ $key | quote }}={{ $value }}
```

Comment:
Don't you want to quote the whole text instead of just the `key` maybe something like this?
```
{{ printf "--%s=%s" $key $value | quote}}
```

Reply:
Done.